keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -263,34 +280,52 @@ is scoped to.
     
       It will cause a lot of keywords to be reserved and users have to memorize those variable when writing rules.
     - Using other names like `this`, `me`, `value`, `_`. The name should be self-explanatory, less chance of conflict and easy to be picked up.
-- For OpenAPIv3 object types, the expression will have direct access to all the
-fields of the object the validator is scoped to.
+
+- For OpenAPIv3 object types, the expression may use field selection to access all the
+  properties of the object the validator is scoped to, e.g. `self.field == 10`.
   
 - For OpenAPIv3 scalar types (integer, string & boolean), the expression will have access to the
-scalar data element the validator is scoped to. The data element will be accessible to CEL
-expressions via `self`, e.g. `len(self) > 10`.
+  scalar data element the validator is scoped to. The data element will be accessible to CEL
+  expressions via `self`, e.g. `len(self) > 10`.
```

Comment:
Are you planning on supporting a `len` function? Totally reasonable in CEL, but I'd use that everywhere in the doc consistently. If this is just an example, then prefer `self.size() > 10`

Reply:
+1 to using the actually available `size()`

Reply:
Thanks, fixing. This is just golang leaking through 😐